### DISCUSSION TITLE
Remix and firebase hosting

### DISCUSSION THREAD

**Discussion by marcuasc** [remix-run/remix#7289] [Q&A] [answered]
Hi, Does anyone have experience with Remix and firebase hosting?
I only get the default index.html from the firebase-template, and its not displaying the application. 

Others have the same problem: https://github.com/remix-run/remix/discussions/6834#discussion-5402543

**Comment by kiliman** [comment, marked as answer]
Check out Remix Guide. I searched for "firebase" and found several links to articles, projects, etc.

This one looked promising: https://github.com/GuptaSiddhant/remix-firebase-template

https://remix.guide/resources/dBcO89A74ruV?q=firebase&sort=top

**Reply by marcuasc** [reply]
Yes, ive seen some similar examples. Is it possible to do it without using firebase functions, and only simple firebase hosting?

**Reply by kiliman** [reply]
Hmm... Remix requires some sort of "server" to execute loaders and actions. Firebase hosting is designed for static sites. If you're not using Cloud Functions, then you won't really be able to use Remix. You might be better off just using React Router then.
